.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
 addons:
   apt:
     packages:
-    - oracle-java8-set-default
+    - oracle-java9-set-default
```

Comment:
revert before moving to merge ready

Reply:
@caseylitton Done. Thanks for the catch.